Sources/KSCrashRecording/KSCrash.m

```diff
@@ -170,8 +170,9 @@ - (void)setUserInfo:(NSDictionary *)userInfo
         }
     }
 
-    const char *userInfoCString = userInfoJSON ? [userInfoJSON bytes] : NULL;
-    kscrash_setUserInfoJSON(userInfoCString);
+    NSString *userInfoString =
+        userInfoJSON ? [[NSString alloc] initWithData:userInfoJSON encoding:NSUTF8StringEncoding] : nil;
```

Comment:
In this case, the problem remains, because `[user Info JSON bytes]` does not return a null terminated string, but `stringWithUTF8String` expects a null terminated string.